.github/workflows/build-and-deploy-images.yml

```diff
@@ -3,6 +3,8 @@ name: Deploy to Docker Hub
 # https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on
 on:
   push:
+    tags:
+      - "v*"
```

Comment:
if GHA has a regex way to do this without the `v` then it would simply the process a little by not requiring one tag with the `v` and another tag without...

Reply:
[GitHub Actions filter syntax](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) has several regex-like features like being able to write `tags: v[1-9]+.[0-9]+.[0-9]+`, but it doesn't have capture groups. Also notice that the `.` was not escaped: these are not regular expressions either. We can't write `v(.*)` and capture the non-v-prefixed version of the release number, so we have to do it later. (I'll have a comment at that point, too).